### PR TITLE
CE-3626: Distinguish users

### PR DESCRIPTION
**Pull request by lukaszkonieczny**
Distinguish newly signed up user from returning user without edit

### CODE REVIEW

File: extensions/wikia/SpitfiresContributionExperiments/Hooks.class.php [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+		return true;
+	}
+
+	private function manageUserActivityGroupCookie( \User $user ) {
+		$newlyRegistered = isset( $_COOKIE[ self::NEWLY_REGISTERED_USER ] );
+
+		if ( !$newlyRegistered ) {
+			$userEditCount = $user->getEditCount();
+			$userWithoutEditCookie = empty( $_COOKIE[ self::WITHOUT_EDIT_USER ] );
```
Comment (Grunny) [inline extensions/wikia/SpitfiresContributionExperiments/Hooks.class.php:54]: Minor: `$userWithoutEditCookie` still reads to me as this should be true if they have the "without edit cookie" where it currently has the opposite value.

File: extensions/wikia/SpitfiresContributionExperiments/Hooks.class.php [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+	public function onUserLoadFromHeliosToken( \User $user ) {
+		$this->manageUserActivityGroupCookie( $user );
+
+		return true;
+	}
+
+	public function onBeforePageDisplay( \OutputPage $out, \Skin $skin ) {
+		if ( isset( $_COOKIE[ self::NEWLY_REGISTERED_USER ] ) || isset( $_COOKIE[ self::WITHOUT_EDIT_USER ] ) ) {
+			$out->addScriptFile('/extensions/wikia/SpitfiresContributionExperiments/scripts/experiments-tracker.js');
+			$out->addScriptFile('/extensions/wikia/SpitfiresContributionExperiments/scripts/my-profile-tour.js');
```
Comment (Grunny) [inline extensions/wikia/SpitfiresContributionExperiments/Hooks.class.php:43]: Minor: Should be spacey per conventions.

**Comment by Grunny** [comment]
Minor comments, otherwise LGTM.

**Comment by lukaszkonieczny** [comment]
I'm closing this PR as it's merged to https://github.com/Wikia/app/pull/10280 and all further work will be continue there

**Pull request closed without merging**